src/inc/msquic.h

```diff
@@ -1083,6 +1084,7 @@ QUIC_STATUS
 typedef enum QUIC_LISTENER_EVENT_TYPE {
     QUIC_LISTENER_EVENT_NEW_CONNECTION      = 0,
     QUIC_LISTENER_EVENT_STOP_COMPLETE       = 1,
+    QUIC_LISTENER_EVENT_DOS_MODE = 2,
```

Comment:
```suggestion
    QUIC_LISTENER_EVENT_DOS_MODE_CHANGED = 2,
```
Also, please align the `=` with the other two.